css-pseudo-classes/style.css

```diff
@@ -0,0 +1,19 @@
+button {
+  height: 40px;
+  width: 110px;
+  margin: 0px;
+}
+
+button:hover {
+  background-color: black;
+  color: white;
+}
```

Comment:
Don't forget to change the user's cursor:

---

![image](https://user-images.githubusercontent.com/16143771/167932907-9ef3cfac-06b4-4a7c-9688-82db09e9be51.png)

Reply:
ohh i didn't see that